MAINTAINERS.md

```diff
@@ -0,0 +1,17 @@
+# Maintainers 
+
+This file lists how the Operation Code Front End project is maintained. When making changes to the system, this file tells you who needs to review your contribution - you need a simple majority of maintainers for the relevant subsystems to provide a 👍 on your pull request. Additionally, you need to not receive a veto from a Lieutenant or the Project Lead.
+
+Check out [how Operation Code Open Source projects are maintained](https://github.com/OperationCode/START_HERE/blob/61cebc02875ef448679e1130d3a68ef2f855d6c4/open_source_maintenance_policy.md) for details on the process, how to become a maintainer, lieutenant, or the project lead.
+
+# Project Lead
+
+* [Kyle Holmberg](http://www.github.com/kylemh)
+
+# Lieutenant
```

Comment:
I would prefer SGT here, it's more semantic (officers don't actually _do_ anything)

Reply:
Can confirm